Assets/Scripts/UI/Menu's/MenuController.cs

```diff
@@ -3,19 +3,73 @@
 
 public sealed class MenuController : MonoBehaviour
 {
+    [Header("Main menu")]
     [SerializeField] private string mainScene;
     [SerializeField] private GameObject activeExplanationMenu;
     [SerializeField] private GameObject activeMainMenu;
 
+    [Header("Pause menu")] 
+    [SerializeField] private GameObject activePauseMenu;
+    [Space]
+    [SerializeField] private GameObject activePauseScreen;
+    [SerializeField] private GameObject activeOptionsScreen;
+    
+    private bool _setPauseMenu = true;
+
     public void ToggleToMainMenuFromExplanation(bool goingToExplanation)
     {
         activeMainMenu.SetActive(!goingToExplanation);
         activeExplanationMenu.SetActive(goingToExplanation);
     }
+
+    public void ToggleToPauseScreenFromOptions(bool goingToOptions)
+    {
+        activePauseScreen.SetActive(!goingToOptions);
+        activeOptionsScreen.SetActive(goingToOptions);
+    }
+
+    private void Start()
+    {
+        activePauseScreen.SetActive(false);
+    }
+
+    private void Update()
+    {
+        SetPauseMenu();
+    }
+    
+    private void SetPauseMenu()
```

Comment:
Vind het gebruik van Set hier niet zo lekker.
Ik geef je een paar voorbeelden dat misschien beter past. Voorkeur is de eerste.

```suggestion
    private void UpdatePauseMenu()
```
```suggestion
    private void ChangePauseMenu()
```
```suggestion
    private void PauseMenu()
```